### PR TITLE
feat: SvelteKit update

### PR DESCRIPTION
**Pull request by joshnuss**
SvelteKit made some routing changes. This PR makes the neccisary updates:

- `post` in a server endpoint is now called `POST`
- endpoint handlers take an object, and the `request` should be destructured, ie `handler({request})`
- endpoint handlers now return a `Response()` object (from [`undici`](https://undici.nodejs.org/))
- to get text of request, call `await request.text()`
- default url for vite in development mode is `localhost:5173`
- updated link to docs (it changed in latest docs)

## TODO
- [x] Update svelte-kit project in examples folder

### CODE REVIEW

File: src/sveltekit.ts [hunk 1]
```diff
@@ -10,11 +10,20 @@ import { registerDevelopmentDefaults } from "./client/config";
 export { Job, EnqueueJobOptions, DefaultJobOptions, QuirrelJobHandler };
 
 registerDevelopmentDefaults({
-  applicationBaseUrl: "localhost:3000",
+  applicationBaseUrl: "localhost:5173",
 });
 
+declare global {
+  interface Response {
+  }
```
Comment (joshnuss) [inline src/sveltekit.ts:18]: Not sure how to do declare a global constructor? `Response` is available in global context in SvelteKit apps.

**Comment by Skn0tt** [comment]
oh wow, thx @joshnuss for noticing the changes + making this PR! I'll look into the global bit.

**Comment by Skn0tt** [comment]
at least for me locally, it looked like `Response` is already available globally as a type. I removed the declaration (https://github.com/quirrel-dev/quirrel/pull/1082/commits/1a1d1b9b76d61dce80527b021ee607cf0fd82077), let's see if this also works in CI :D

**Comment by Skn0tt** [comment]
@all-contributors please add @joshnuss for code, docs

**Pull request merged**